README.md

```diff
@@ -463,3 +463,20 @@ cmake -DCMAKE_BUILD_TYPE=Release -DBENCHMARK_ENABLE_LTO=true
 
 ## Linking against the library
 When using gcc, it is necessary to link against pthread to avoid runtime exceptions. This is due to how gcc implements std::thread. See [issue #67](https://github.com/google/benchmark/issues/67) for more details.
+
+
+## Running a subset of the benchmarks
+
+Similar to the `--gtest_filter` functionality of `googletest`, `benchmark` has a `--benchmark_filter` option:
```

Comment:
It's not so similar because the two take different kinds of regexp.  Please document this.